docs/static/security/basic-auth.asciidoc

```diff
@@ -0,0 +1,82 @@
+[discrete]
```

Comment:
Ultimately, the filename doesn't really matter, but the anchor (Line 2) does--not to say that we can't change either or both later.  The "Basic authentication" terminology aligns with Beats](https://www.elastic.co/guide/en/beats/filebeat/master/securing-communication-elasticsearch.html), while [Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/master/manually-configure-security.html#security-basic-overview) has moved to  the phrase "basic security." Does using "basic security" instead of "basic auth" address your concerns?